app/webpack/taxa/show/components/status_tab.jsx

```diff
@@ -141,6 +141,17 @@ const StatusTab = ( { statuses, listedTaxa, listedTaxaCount } ) => {
                       text={status.description}
                     />
                   ) }
+                  { status.taxon_id && status.taxon_name && status.taxon_id !== taxonId && (
+                      <div
+                          className="text-muted"
+                          dangerouslySetInnerHTML={{
+                            __html: I18n.t( "status_applied_from_higher_level_taxon_html", {
+                              url: `/taxa/${status.taxon_id}`,
+                              taxon: `${status.taxon_name}`
```

Comment:
This will render the taxon name without any formatting and without regard to the viewer's name preferences, which I think is fine, but if you want to get ahead of the inevitable complaints, I think you'd want to:

1) Pass a taxon to this component instead of the `taxonId` prop
2) Look up the taxon record for this status in `taxon.ancestors`
3) Use `ReactDOMServer.renderToString()` to render a `<SplitTaxon>` for that taxon and pass that into `I18n.t`

Anyway, if you're up for that, great, but I'm fine merging this without that too.